apollo-ios/Sources/Apollo/ApolloClient.swift

```diff
@@ -130,13 +134,21 @@ public final class ApolloClient: ApolloClientProtocol, Sendable {
 
   public func fetch<Query: GraphQLQuery>(
     query: Query,
-    cachePolicy: CachePolicy? = nil
+    cachePolicy: CachePolicy? = nil,
```

Comment:
What's meant to be on the next line after the new `,`?

Reply:
This was a mistake, but these APIs are changed in #684, so this isn't an issue.